### PR TITLE
Add remaining packet types

### PR DESCRIPTION
**Pull request by adryzz**
Adds all the remaining packet types.

All the `Packet<T>.ValidateAs()` that aren't implemented in MediapipeUnity throw a `NotSupportedException`.

Currently waiting for the next protofiles to be added to master.

- [x] Packets
- [x] Packet unit tests

### CODE REVIEW

File: src/Akihabara/Framework/Packet/LandmarkListVectorPacket.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        }
+
+        public override StatusOr<List<LandmarkList>> Consume()
+        {
+            throw new NotSupportedException();
+        }
+
+        public override Status ValidateAsType()
+        {
+            throw new NotImplementedException();
```
Comment (Speykious) [inline src/Akihabara/Framework/Packet/LandmarkListVectorPacket.cs:36]: I think this should be a `NotSupportedException()` like all the others

**Review by Speykious — CHANGES_REQUESTED** [review]
Besides this one thing, I think everthing is fine... I'm really not sure about the unsupported stuff. Surely there must be corresponding bindings right? But in any case we have that I guess.

Are there any unit tests for these packets to port over as well?

File: src/Akihabara/Framework/Packet/LandmarkListVectorPacket.cs (hunk 1 above)
Comment (adryzz) [reply src/Akihabara/Framework/Packet/LandmarkListVectorPacket.cs:36]: oh, my bad

**Comment by Speykious** [comment]
*Ah yes, `fix issue` the descriptivest of commits* :p

**Comment by adryzz** [comment]
> _Ah yes, `fix issue` the descriptivest of commits_ :p

i mean... it's directly below your issue

**Comment by adryzz** [comment]
What we need now to finish this PR are the `FaceGeometry`, `EglSurfaceHolder` and `TimedModelMatrixProtoList` proto files

**Comment by Speykious** [comment]
Isn't Face Geometry a module though?

**Comment by adryzz** [comment]
> Isn't Face Geometry a module though?

i don't know exactly, as i haven't looked. Still though, we need those types ported. I got the packets and the tests ready for commit

**Comment by adryzz** [comment]
Ready for review.

Also, this will have to be documented on the wiki or README. 

`EglSurfaceHolder` and all the relative classes only build on Linux and are meant to be used for OpenGL ES only (so mobile).

They will be excluded from compilation unless the person compiling #defines `OPENGL_ES` with preprocessor directives, kinda like how homuler did it (he used Unity specific #defines, that we obviously don't have), so this is to have a 1:1 port and no build issues in the future.

File: src/Akihabara.Tests/Framework/Packet/EglSurfaceHolderPacketTest.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Akihabara.Tests.Framework.Packet
+{
+#if OPENGL_ES
+    [TestFixture]
+    public class EglSurfaceHolderPacketTest
+    {
+#region Constructor
```
Comment (Speykious) [inline src/Akihabara.Tests/Framework/Packet/EglSurfaceHolderPacketTest.cs:19]: Quick styling inconsistency: some `#region`s start at the beginning of the line, while some start at some level of indentation.

**Review by Speykious — CHANGES_REQUESTED** [review]
Almost everything is fine, besides a quick styling thing.
The next step after merging the PR will be to make actual examples with Face Mesh graphs and others!

File: src/Akihabara.Tests/Framework/Packet/EglSurfaceHolderPacketTest.cs (hunk 2 above)
Comment (adryzz) [reply src/Akihabara.Tests/Framework/Packet/EglSurfaceHolderPacketTest.cs:19]: I'll try fixing it now, and if i can't, I'll do that when i come home

**Review by Speykious — APPROVED** [review]
Perfect :)

**Comment by Speykious** [comment]
I can sign the commit

**Comment by adryzz** [comment]
i already committed but if you can, do that

**Review by Speykious — APPROVED** [review]
Alright, we can finally safely merge this PR!

**Pull request merged**